.github/workflows/build-gcp-guest-agent.yml

```diff
@@ -100,7 +100,7 @@ jobs:
       - name: Log in to the Container registry
         id: docker-login
         if: steps.needs-build.outputs.out == 'true'
-        uses: ./.github/actions/container_registry_login
+        uses: ./constellation/.github/actions/container_registry_login
```

Comment:
I don't get this change. We use the old version everywhere else.

Reply:
We're checking out side-by-side with guest-agent source, see https://github.com/edgelesssys/constellation/blob/feat/ci/gcpguestagent/.github/workflows/build-gcp-guest-agent.yml#L82

Reply:
Ah I see, thanks for the pointer.

Reply:
Bug introduced in https://github.com/edgelesssys/constellation/commit/025d34a259cff3a0fca8d219ffab3b3ef7abbe8c